diff -Nru m4-1.4ppre2.orig/configure.in m4-1.4ppre2/configure.in
--- m4-1.4ppre2.orig/configure.in	Sun Jan 16 17:28:03 2000
+++ m4-1.4ppre2/configure.in	Thu Apr  3 09:43:12 2003
@@ -136,18 +136,10 @@
 
 dnl AC_LINK_FILES($nls_cv_header_libgt, $nls_cv_header_intl)
 
-LTLIBOBJS=`echo "$LIBOBJS" | sed 's,.o ,.lo ,g;s,.o$,.lo,'`
+LIB@&t@OBJS=`echo "$LIB@&t@OBJS" | sed 's,\.[[^.]]* ,$U&,g;s,\.[[^.]]*$,$U&,'`
+LTLIBOBJS=`echo "$LIB@&t@OBJS" | sed 's,\.[[^.]]* ,.lo ,g;s,\.[[^.]]*$,.lo,'`
 AC_SUBST(LTLIBOBJS)
 
-# This no-op line works around a bug in automake 1.4:
-# it doesn't recognize the assignment to LIBOBJS
-# in acm4/malloc.m4 and acm4/realloc.m4.
-# : LIBOBJS="$LIBOBJS malloc.o realloc.o"
-
-# This is necessary so that .o files in LIBOBJS are also built via
-# the ANSI2KNR-filtering rules.
-LIBOBJS=`echo $LIBOBJS|sed 's/\.o /\$U.o /g;s/\.o$/\$U.o/'`
-
 AC_OUTPUT(Makefile acm4/Makefile intl/Makefile po/Makefile.in doc/Makefile \
 lib/Makefile src/Makefile modules/Makefile tests/Makefile examples/Makefile, \
 [sed -e "/POTFILES =/r po/POTFILES" po/Makefile.in > po/Makefile; \
