--- m4-1.4n/configure.in.orig	Fri Sep  5 00:42:01 2003
+++ m4-1.4n/configure.in	Fri Sep  5 00:42:30 2003
@@ -38,7 +38,7 @@
 AC_FUNC_ALLOCA
 AC_FUNC_VPRINTF
 AC_REPLACE_FUNCS(strtol obstack xmalloc xstrdup)
-AC_CHECK_FUNC(getopt_long, , [LIBOBJS="$LIBOBJS getopt1.o getopt.o"])
+AC_CHECK_FUNC(getopt_long, , [AC_LIBOBJ(getopt1); AC_LIBOBJ(getopt)])
 jm_WITH_REGEX
 jm_PREREQ_ERROR
 
--- m4-1.4n/acinclude.m4~	Fri Sep  5 00:51:01 2003
+++ m4-1.4n/acinclude.m4	Fri Sep  5 00:51:01 2003
@@ -25,7 +25,7 @@
 		    jm_with_regex=$withval,
 		    jm_with_regex=yes)
 	if test "$jm_with_regex" = yes; then
-	  LIBOBJS="$LIBOBJS regex.o"
+	  AC_LIBOBJ([regex])
 	fi
       ],
     )
--- m4-1.4n/aclocal.m4~	Fri Sep  5 00:51:02 2003
+++ m4-1.4n/aclocal.m4	Fri Sep  5 00:51:02 2003
@@ -38,7 +38,7 @@
 		    jm_with_regex=$withval,
 		    jm_with_regex=yes)
 	if test "$jm_with_regex" = yes; then
-	  LIBOBJS="$LIBOBJS regex.o"
+	  AC_LIBOBJ([regex])
 	fi
       ],
     )
