--- m4-1.4ppre2/configure.in.orig	Sun Jan 16 17:28:03 2000
+++ m4-1.4ppre2/configure.in	Fri Apr  4 05:53:47 2003
@@ -2,19 +2,12 @@
 # Copyright (C) 1991-2000 Free Software Foundation, Inc.
 # Process this file with autoconf to produce a configure script.
 
-undefine([changeword])
-
 AC_INIT(src/m4.c)
 AM_CONFIG_HEADER(config.h)
 AC_ARG_PROGRAM
 
 AM_INIT_AUTOMAKE(m4,1.4ppre2)
 
-ALL_LINGUAS="cs de el fr it ja nl pl ru sv"
-
-pkglibexecdir='${libexecdir}'/$PACKAGE
-AC_SUBST(pkglibexecdir)
-
 AC_PROG_AWK
 AC_PROG_CC
 AC_PROG_INSTALL
@@ -57,14 +50,21 @@
 AC_FUNC_VPRINTF
 AC_REPLACE_FUNCS(strtol xmalloc xstrdup)
 M4OBJS=
-AC_CHECK_FUNC(getopt_long, , [M4OBJS="getopt1.o getopt.o"])
+AC_CHECK_FUNC(getopt_long, [], [M4OBJS="getopt1.o getopt.o"])
 AC_SUBST(M4OBJS)
-jm_PREREQ_ERROR
+AC_CHECK_FUNCS(strerror strerror_r vprintf doprnt)
+AC_HEADER_STDC
 
 AC_MSG_CHECKING(ecvt declaration)
 AC_EGREP_HEADER(ecvt, stdlib.h,
-  [AC_MSG_RESULT(yes); AC_DEFINE(HAVE_EFGCVT, 2)],
-  [AC_MSG_RESULT(no); AC_CHECK_FUNCS(ecvt)])
+	[AC_MSG_RESULT(yes)
+	 AC_DEFINE(HAVE_EFGCVT, 2, 
+		[Define to 1 if you have ecvt(3), fcvt(3) and gcvt(3),
+		 define to 2 if these are declared in<stdlib.h>.])
+	],
+	[AC_MSG_RESULT(no)
+	 AC_CHECK_FUNCS(ecvt)]
+)
 
 AC_MSG_CHECKING(if stack overflow is detectable)
 # Code from Jim Avera <jima@netcom.com>.
@@ -88,29 +88,33 @@
 fi
 AC_MSG_RESULT($use_stackovf)
 if test "$use_stackovf" = yes; then
-  AC_DEFINE(USE_STACKOVF)
-  STACKOVF=stackovf.${U}o
-  AC_SUBST(STACKOVF)
-  AC_EGREP_HEADER(rlim_t, sys/resource.h, , AC_DEFINE(rlim_t, int))
-  AC_EGREP_HEADER(stack_t, signal.h, , AC_DEFINE(stack_t, struct sigaltstack))
-  AC_EGREP_HEADER(sigcontext, signal.h, AC_DEFINE(HAVE_SIGCONTEXT))
-  AC_EGREP_HEADER(siginfo_t, signal.h,
-		  AC_DEFINE(HAVE_SIGINFO_T, 1,
-			    [Define if <signal.h> declares siginfo_t. ]))
-
-  AC_TRY_COMPILE([#include <signal.h>],
-	    [struct sigaltstack x; x.ss_base = 0;],
-	    AC_DEFINE(ss_sp, ss_base))
+	AC_DEFINE(USE_STACKOVF, 1, [Define if using stack overflow detection.])
+	STACKOVF=stackovf.${U}o
+	AC_SUBST(STACKOVF)
+	AC_EGREP_HEADER(rlim_t, sys/resource.h, [],
+		AC_DEFINE(rlim_t, int,
+			[Define to int if rlim_t is not defined in sys/resource.h]))
+	AC_EGREP_HEADER(stack_t, signal.h, [],
+		AC_DEFINE(stack_t, structsigaltstack,
+			[Define to struct sigaltstack if stack_t is not defined in sys/signal.h]))
+	AC_EGREP_HEADER(sigcontext, signal.h, AC_DEFINE(HAVE_SIGCONTEXT))
+	AC_EGREP_HEADER(siginfo_t, signal.h,
+		AC_DEFINE(HAVE_SIGINFO_T, 1,
+			[Define if <signal.h> declares siginfo_t.]))
+
+	AC_TRY_COMPILE([#include <signal.h>],
+		[struct sigaltstack x; x.ss_base = 0;],
+		AC_DEFINE(ss_sp, ss_base))
 fi
 
 AC_MSG_CHECKING(if changeword is wanted)
 AC_ARG_ENABLE(changeword,
 [  --enable-changeword     enable -W and changeword() builtin],
 [if test "$enableval" = yes; then
-  AC_MSG_RESULT(yes)
-  AC_DEFINE(ENABLE_CHANGEWORD)
+	AC_MSG_RESULT(yes)
+	AC_DEFINE(ENABLE_CHANGEWORD, 1, [Define if the changeword(REGEXP) functionnality is wanted.])
 else
-  AC_MSG_RESULT(no)
+	AC_MSG_RESULT(no)
 fi], [AC_MSG_RESULT(no)])
 
 AM_WITH_MODULES
@@ -121,37 +125,25 @@
 AC_CHECK_FUNCS(getcwd gethostname mktime uname)
 AC_CHECK_FUNCS(setenv unsetenv putenv clearenv)
 
-
 AM_WITH_GMP
 
 AM_WITH_DMALLOC
+ALL_LINGUAS="cs de el fr it ja nl pl ru sv"
 AM_GNU_GETTEXT
 
-if test "$USE_INCLUDED_LIBINTL" = yes; then
-  INTLINCL='-I$(top_srcdir)/intl'
-
-  AC_SUBST(INTLINCL)
-fi
-
-
-dnl AC_LINK_FILES($nls_cv_header_libgt, $nls_cv_header_intl)
-
-LTLIBOBJS=`echo "$LIBOBJS" | sed 's,.o ,.lo ,g;s,.o$,.lo,'`
+LIB@&t@OBJS=`echo "$LIB@&t@OBJS" | sed 's,\.[[^.]]* ,$U&,g;s,\.[[^.]]*$,$U&,'`
+LTLIBOBJS=`echo "$LIB@&t@OBJS" | sed 's,\.[[^.]]* ,.lo ,g;s,\.[[^.]]*$,.lo,'`
 AC_SUBST(LTLIBOBJS)
 
-# This no-op line works around a bug in automake 1.4:
-# it doesn't recognize the assignment to LIBOBJS
-# in acm4/malloc.m4 and acm4/realloc.m4.
-# : LIBOBJS="$LIBOBJS malloc.o realloc.o"
-
-# This is necessary so that .o files in LIBOBJS are also built via
-# the ANSI2KNR-filtering rules.
-LIBOBJS=`echo $LIBOBJS|sed 's/\.o /\$U.o /g;s/\.o$/\$U.o/'`
-
-AC_OUTPUT(Makefile acm4/Makefile intl/Makefile po/Makefile.in doc/Makefile \
-lib/Makefile src/Makefile modules/Makefile tests/Makefile examples/Makefile, \
-[sed -e "/POTFILES =/r po/POTFILES" po/Makefile.in > po/Makefile; \
-sh $srcdir/tests/mkconfig.sh; \
-test -z "$CONFIG_HEADERS" || echo timestamp > stamp-h])
-
-
+AC_OUTPUT([
+	Makefile
+	acm4/Makefile
+	intl/Makefile
+	po/Makefile.in
+	doc/Makefile
+	lib/Makefile
+	src/Makefile
+	modules/Makefile
+	tests/Makefile
+	examples/Makefile],
+	[sh $srcdir/tests/mkconfig.sh])
--- m4-1.4ppre2/src/Makefile.am.orig	Sun Jan 16 17:49:03 2000
+++ m4-1.4ppre2/src/Makefile.am	Sat Apr  5 15:58:22 2003
@@ -1,31 +1,16 @@
 AUTOMAKE_OPTIONS = ansi2knr
 
-CLEANFILES = m4@EXEEXT@
-MAINTAINERCLEANFILES = Makefile.in ansi2knr.c ansi2knr.1 pathconf.h
-
-AM_CPPFLAGS = @LIBM4_DLL_IMPORT@
-INCLUDES = -I$(top_srcdir)/lib @INTLINCL@
+INCLUDES = -I$(top_srcdir)/lib
+AM_CPPFLAGS = $(LIBM4_DLL_IMPORT) -DMODULE_PATH=\"$(pkglibdir)\"
   
-CFLAGS = @CFLAGS@
-
-LIBS = @LIBS@
-
-MODULE_PATH = @MODULE_PATH@
-
 noinst_HEADERS = getopt.h ltdl.h
 
 bin_PROGRAMS = m4
 m4_SOURCES = m4.c m4.h builtin.c debug.c eval.c evalmp.c \
 	format.c freeze.c input.c macro.c output.c path.c symtab.c \
-	stackovf.c module.c ltdl.c getopt.c getopt1.c
-EXTRA_m4_SOURCES = numb.c numb.h
-
-
-m4_LDADD = @M4OBJS@ $(top_srcdir)/lib/libm4.la @INTLLIBS@
-m4_LDFLAGS = -no-undefined @DLLDFLAGS@
-
-$(bin_PROGRAMS): @M4OBJS@
-path.o: path.c pathconf.h
+	stackovf.c module.c
+EXTRA_m4_SOURCES = numb.c numb.h ltdl.c getopt.c getopt1.c
 
-pathconf.h: Makefile
-	@echo "#define MODULE_PATH \"${MODULE_PATH}\"" >pathconf.h
+m4_LDADD = $(M4OBJS) $(top_srcdir)/lib/libm4.la $(LTLIBINTL) -lltdl
+m4_LDFLAGS = -no-undefined $(DLLDFLAGS)
+ 
\ No newline at end of file
--- m4-1.4ppre2/src/module.c~	Fri Apr  4 05:48:27 2003
+++ m4-1.4ppre2/src/module.c	Fri Apr  4 05:48:27 2003
@@ -23,7 +23,6 @@
 
 #define MODULES_UNINITIALISED -4444
 
-#include "pathconf.h"
 #include "ltdl.h"
 
 #define DEBUG_MODULES  /* Define this to see runtime debug info. */
--- m4-1.4ppre2/lib/Makefile.am~	Wed Apr  2 18:51:59 2003
+++ m4-1.4ppre2/lib/Makefile.am	Wed Apr  2 18:51:59 2003
@@ -2,13 +2,12 @@
 AUTOMAKE_OPTIONS = gnu ../src/ansi2knr
 MAINTAINERCLEANFILES = Makefile.in
 
-LDFLAGS = -no-undefined -export-dynamic @DLLDFLAGS@
+AM_LDFLAGS = -no-undefined -export-dynamic $(DLLDFLAGS)
 
-INCLUDES = @INTLINCL@
 include_HEADERS = m4module.h m4error.h m4obstack.h m4regex.h
 noinst_HEADERS = m4private.h
 
 lib_LTLIBRARIES = libm4.la
 libm4_la_SOURCES = m4module.c m4error.c m4obstack.c m4regex.c
-libm4_la_LIBADD = @LTLIBOBJS@
+libm4_la_LIBADD = $(LTLIBOBJS)
 
--- m4-1.4ppre2/modules/Makefile.am~	Wed Apr  2 18:53:49 2003
+++ m4-1.4ppre2/modules/Makefile.am	Wed Apr  2 18:53:49 2003
@@ -1,6 +1,6 @@
 MAINTAINERCLEANFILES = Makefile.in
 
-pkglibexec_LTLIBRARIES = test.la shadow.la time.la stdlib.la
+pkglib_LTLIBRARIES = test.la shadow.la time.la stdlib.la
 
 EXTRA_DIST = test.m4 shadow.m4 time.m4 time2.m4 stdlib.m4 \
 	 $(TESTS) defs
@@ -8,8 +8,8 @@
 TESTS = test.test time.test shadow.test \
 	modpath1.test modpath2.test modpath3.test modpath4.test
 
-AM_CPPFLAGS = @LIBM4_DLL_IMPORT@
-INCLUDES = -I$(top_srcdir)/src -I$(top_srcdir)/lib -I$(top_srcdir)/libltdl -I/usr/local/include @INTLINCL@
+AM_CPPFLAGS = $(LIBM4_DLL_IMPORT)
+INCLUDES = -I$(top_srcdir)/src -I$(top_srcdir)/lib -I$(top_srcdir)/libltdl $(INTLINCL)
  
 test_la_SOURCES = test.c
 shadow_la_SOURCES = shadow.c
@@ -17,7 +17,7 @@
 stdlib_la_SOURCES = stdlib.c
 
 LIBS = $(top_srcdir)/lib/libm4.la
-LDFLAGS = -no-undefined -module -avoid-version @DLLDFLAGS@
+AM_LDFLAGS = -no-undefined -module -avoid-version $(LLDFLAGS)
 
 test_la_LDFLAGS = -module
 shadow_la_LDFLAGS = -module
--- m4-1.4ppre2/doc/Makefile.am~	Wed Apr  2 18:54:39 2003
+++ m4-1.4ppre2/doc/Makefile.am	Wed Apr  2 18:54:39 2003
@@ -2,12 +2,10 @@
 
 man_MANS = m4.1
 
-EXTRA_DIST = helptoman.pl # $(MANS)
+EXTRA_DIST = helptoman.pl
 
 MAINTAINERCLEANFILES = Makefile.in mdate-sh texinfo.tex m4.1
 
-PERL = @PERL@
-
 %.1: helptoman.pl ../src/$*
 	@if test -f ../src/$*; then \
 	  echo "Updating the \`man' page \`$@'"; \
