diff -urN m4-1.4.8.org/doc/m4.texinfo m4-1.4.8/doc/m4.texinfo
--- m4-1.4.8.org/doc/m4.texinfo	2006-11-20 14:55:43.000000000 +0100
+++ m4-1.4.8/doc/m4.texinfo	2006-11-26 03:51:12.974022250 +0100
@@ -56,9 +56,9 @@
 @end quotation
 @end copying
 
-@dircategory GNU programming tools
+@dircategory GNU programming tools:
 @direntry
-* M4: (m4).                     A powerful macro processor.
+* M4: (m4).				A powerful macro processor.
 @end direntry
 
 @titlepage
