diff -Nru m4-1.4n/doc/m4.texinfo m4-1.4n.new/doc/m4.texinfo
--- m4-1.4n/doc/m4.texinfo	Sun Nov 29 02:37:54 1998
+++ m4-1.4n.new/doc/m4.texinfo	Sun Jan  3 16:28:16 1999
@@ -11,9 +11,10 @@
 
 @ifinfo
 @format
-START-INFO-DIR-ENTRY
-* m4: (m4).			A powerful macro processor.
-END-INFO-DIR-ENTRY
+@dircategory Miscellaneous:
+@direntry
+* m4: (m4)				A powerful macro processor
+@end direntry
 @end format
 @end ifinfo
 
