--- m4-1.4ppre2/src/Makefile.am.wiget	Thu Mar  6 22:48:32 2003
+++ m4-1.4ppre2/src/Makefile.am	Thu Mar  6 22:50:23 2003
@@ -6,10 +6,6 @@
 AM_CPPFLAGS = @LIBM4_DLL_IMPORT@
 INCLUDES = -I$(top_srcdir)/lib @INTLINCL@
   
-CFLAGS = @CFLAGS@
-
-LIBS = @LIBS@
-
 MODULE_PATH = @MODULE_PATH@
 
 noinst_HEADERS = getopt.h ltdl.h
@@ -17,11 +13,11 @@
 bin_PROGRAMS = m4
 m4_SOURCES = m4.c m4.h builtin.c debug.c eval.c evalmp.c \
 	format.c freeze.c input.c macro.c output.c path.c symtab.c \
-	stackovf.c module.c ltdl.c getopt.c getopt1.c
+	stackovf.c module.c
 EXTRA_m4_SOURCES = numb.c numb.h
 
 
-m4_LDADD = @M4OBJS@ $(top_srcdir)/lib/libm4.la @INTLLIBS@ -ldl
+m4_LDADD = @M4OBJS@ $(top_srcdir)/lib/libm4.la @LTLIBINTL@ -lltdl
 m4_LDFLAGS = -no-undefined @DLLDFLAGS@
 
 $(bin_PROGRAMS): @M4OBJS@
